python/packages/ai/teams/app.py

```diff
@@ -202,13 +205,97 @@ def __selector__(context: TurnContext):
 
             return False
 
-        def __call__(func: Callable[[TurnContext, StateT], Awaitable[bool]]):
+        def __call__(func: RouteHandler[StateT]):
+            self._routes.append(Route[StateT](__selector__, func))
+            return func
+
+        return __call__
+
+    def anonymous_query_link(self, command_id: Union[str, Pattern[str]]):
```

Comment:
Do we need to add an extra `message_extension` layer for the functionalities like JS? So the code will be `app.message_extension.anonymous_query_link(...)`?

Reply:
no that layer in JS doesn't do anything its just a class, I think its cleaner to have all our route selectors at the app level to keep it organized since the `message_extension` part isn't really doing anything.